File: Fondamenti_programmazione_II/Java/EserciziEsame/GestioneProgettiRicercaProf/Ricerca.java
package EserciziEsame.GestioneProgettiRicercaProf;

public class Ricerca extends Progetto{
    private int partner;
    private String argomento;

    public Ricerca(int codice, String titolo, String coordinatore, String organizzazione, String argomento, int partner, double importo){
        super(codice,titolo,coordinatore,organizzazione,importo);
        this.partner=partner;
        this.argomento=argomento;
    }

    public String toString(){
        return "ricerca\t"+super.toString()+argomento+"\t"+partner+"\t-\t"+importo*1000;
    }
}

File: Fondamenti_programmazione_II/Python/Esami/Gestione_progetti_ricerca/Ricerca.py
from Progetto import Progetto

class Ricerca(Progetto):
    def __init__(self, codice, titolo, coordinatore, organizzazione, importo, argomento, partner):
        super().__init__(codice, titolo, coordinatore, organizzazione, importo, "ricerca")
        self._argomento= argomento
        self._partner = partner

    def __str__(self):
        return super().__str__() + self._argomento +"\t"+ str(self._partner) +"\t-\t"+ str(self._importo*1000)